.github/workflows/update-external-studies.yml

```diff
@@ -49,7 +49,6 @@ jobs:
       # The PR will still require manual approval, this just reduces it to a one-click process
       - name: Enable automerge
         if: steps.create_pr.outputs.pull-request-operation == 'created'
-        uses: peter-evans/enable-pull-request-automerge@684fed02ccc9b5eefcf7d40b65b3cd44255bd5bc # v2.5.0
-        with:
-          pull-request-number: ${{ steps.create_pr.outputs.pull-request-number }}
-          merge-method: squash
+        run: gh pr merge --merge --auto --squash ${{ steps.create_pr.outputs.pull-request-number }}
+        env:
+          GH_TOKEN: ${{ secrets.PAT }}
```

Comment:
suggestion: If the PAT isn't generic, and is a fine-grained token, with permissions restricted to allow enabling automerge, I'd consider naming it as such.

If it is a generic PAT with no particularly restricted permissions, it may be worth considering restricting them.

Reply:
Ah, right! :+1: